### PR TITLE
removed device word

### PR DESCRIPTION
**Pull request by RAJU2529**
this is my own PR i removed word **Device**

### CODE REVIEW

**Comment by illfated** [comment]
@RAJU2529 : Could you please take me through the reasoning behind your choice to remove the word?

From my point of view, that text mirrors the page title, so I don't see the reason for making this change. Please help me understand.

File: windows/client-management/mdm/healthattestation-csp.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Device HealthAttestation CSP
+title: HealthAttestation CSP
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/healthattestation-csp.md:2]:
```suggestion
title: Device HealthAttestation CSP
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Agree with @illfated

File: windows/client-management/mdm/healthattestation-csp.md (hunk 1 above)
Comment (RAJU2529) [reply windows/client-management/mdm/healthattestation-csp.md:2]: @JohanFreelancer9 . This **HealthAttestation CSP** is mentioned in below article

https://docs.microsoft.com/en-us/windows/client-management/mdm/configuration-service-provider-reference

so thats why I removed device

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**